src/main/java/redis/clients/jedis/json/util/JsonUtils.java

```diff
@@ -0,0 +1,53 @@
+package redis.clients.jedis.json.util;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import redis.clients.jedis.exceptions.JedisException;
+
+public final class JsonUtils {
+
+    private static final Logger logger = LoggerFactory.getLogger(JsonUtils.class);
+
+    private static volatile JsonTools JSON_TOOLS;
+
+    static {
+        JSON_TOOLS = loadGsonTools();
+    }
+
+    private JsonUtils() {
+    }
+
+    public static JsonTools getJsonTools() {
+        return JSON_TOOLS;
+    }
+
+    public static void setJsonTools(JsonTools jsonTools) {
+        JSON_TOOLS = jsonTools;
+    }
+
+    public static String toJson(Object src) {
+        if (JSON_TOOLS == null) {
+            throw new JedisException("GSON found on classpath, please add com.google.code.gson:gson to your classpath or inject your own impl. of JsonTools");
```

Comment:
Shouldn't this say "GSON <ins>not</ins> found on classpath"?

Reply:
thanks!